lua/nvim_lsp/pyls.lua

```diff
@@ -0,0 +1,28 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+local cwd = vim.loop.cwd()
+
+skeleton.pyls = {
+  default_config = {
+    cmd = {"pyls"};
+    filetypes = {"python"};
+    root_dir = function() return cwd end;
+    log_level = lsp.protocol.MessageType.Warning;
+  };
+  -- on_new_config = function(new_config) end;
+  -- on_attach = function(client, bufnr) end;
+  docs = {
+    description = [[
+https://github.com/palantir/python-language-server
+
+python-language-server, a language server for Python
+]];
+    default_config = {
+      root_dir = "vim's starting directory";
```

Comment:
Do you think it would be a good idea to explain why we made this choice? i.e. that it makes only one server per neovim session?

Reply:
To be honest, I've never worried about that -- I just started the server as advised in vim-lsp docs and went about my day. (But then, I'm only working with single-file scripts.)